### DISCUSSION TITLE
Website

### DISCUSSION THREAD

**Discussion by goyalyashpal** [KDE/ghostwriter#653] [Q&A] [answered]
hi, where's the source of the website https://wereturtle.github.io/ghostwriter/ ?
i tried https://github.com/wereturtle/wereturtle.github.io but couldnt dig further.

**Comment by maxicarlos08** [comment, marked as answer]
It's on the `gh-pages` branch of this repository ([link](https://github.com/wereturtle/ghostwriter/tree/gh-pages))

**Reply by goyalyashpal** [reply]
uh oh, i have completely forgotten why did i ask in the first place 😅 

i should have mentioned it or at least given some hint to it in the original comment
